frontend/src/components/PluginDataContextProvider.test.tsx

```diff
@@ -18,22 +18,11 @@ const TestPluginDataContextProvider = (props: ProviderProps<PluginData>) => {
 }
 
 describe('PluginDataContextProvider', () => {
-  it('does not render LoadData until requested', async () => {
-    const pluginData: PluginData = {
-      ...defaultContext,
-      loaded: false,
-      load: jest.fn(),
-    }
-    render(<TestPluginDataContextProvider value={pluginData} />)
-    await waitForText('Loading')
-    expect(pluginData.load).toHaveBeenCalled()
-  })
```

Comment:
This test verified 2 things:
- the `load()` function is called if we're not already loaded
- 'Loading' displays until then

We no longer have the 'Loading' text to display, but we could keep the other part of this test.

Reply:
restored and testing for load()